sdjournal/journal.go

```diff
@@ -245,15 +273,18 @@ var libsystemdFunctions = map[string]unsafe.Pointer{}
 // Journal entry field strings which correspond to:
 // http://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html
 const (
-	SD_JOURNAL_FIELD_SYSTEMD_UNIT      = "_SYSTEMD_UNIT"
-	SD_JOURNAL_FIELD_SYSLOG_IDENTIFIER = "SYSLOG_IDENTIFIER"
-	SD_JOURNAL_FIELD_MESSAGE           = "MESSAGE"
-	SD_JOURNAL_FIELD_PID               = "_PID"
-	SD_JOURNAL_FIELD_UID               = "_UID"
-	SD_JOURNAL_FIELD_GID               = "_GID"
-	SD_JOURNAL_FIELD_HOSTNAME          = "_HOSTNAME"
-	SD_JOURNAL_FIELD_MACHINE_ID        = "_MACHINE_ID"
-	SD_JOURNAL_FIELD_TRANSPORT         = "_TRANSPORT"
+	SD_JOURNAL_FIELD_SYSTEMD_UNIT        = "_SYSTEMD_UNIT"
+	SD_JOURNAL_FIELD_SYSLOG_IDENTIFIER   = "SYSLOG_IDENTIFIER"
```

Comment:
What do you mean? Ease access to other non address fields by resolving them from key values instead of a generic one? Something like:

```
// JournalEntry represents all fields of a journal entry plus address fields.
type JournalEntry struct {
	Fields             map[string]string
	Priority           string?
	Cursor             string
	RealtimeTimestamp  uint64
	MonotonicTimestamp uint64
}
```

WDYT if we leave this for another pull request? If we agree I'll work on that after this one.